# Generated for NaoQi method ALFaceDetection.clearDatabase
# clearDatabase

---
# true if the operation succeeds
bool value


